src/vm/dllimport.cpp

```diff
@@ -5791,8 +5807,15 @@ class LoadLibErrorTracker
         }
     }
 
+    void SetMessage(LPCSTR message)
+    {
+        SString append_string = SString(SString::Utf8, message);
+        m_message.Append(append_string);
```

Comment:
This is just going to jam the strings together isn't it?, you probably want to include \n's.